app/components/Article/Article.module.scss

```diff
@@ -0,0 +1,83 @@
+@use '@/styles/mixins.scss' as mix;
+@use '@/styles/variables.scss' as var;
+
+.article {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid var.$color-700-red;
+  border-radius: var.$border-radius;
+  height: 50vh;
+
+  img {
+    height: 14rem;
+    width: 100%;
+    object-fit: cover;
+    border-top-right-radius: var.$border-radius;
+    border-top-left-radius: var.$border-radius;
+  }
+
+  .info {
```

Comment:
Please replace it with `&__info` because when you put it like that you have 2 class specificity for no reason, and potentially in future if you need to overwrite it you’ll have put 3 classes